packages/store/src/utils/storage.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
We already implement this here: `packages/app/src/systems/Core/utils/storage.ts`. As I notice the code is replicated can we create a package utils, can be done on another issue.

Reply:
I added it as a new package, but didn't change inside the `app` to avoid bloat this pull request more and more 😅 but it's implement in the store, so I can change inside the app in another PR